.github/workflows/scripts/prepare_collection.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -xmveuo pipefail
+
+echo "Installing pulp.pulp_installer collection"
+echo ::group::INSTALL_COLLECTION
+make vendor
+make install
+mkdir -p ~/.ansible/collections/ansible_collections
+cp -r ./build/collections/ansible_collections/* ~/.ansible/collections/ansible_collections/
+echo ::endgroup::
+
+echo "Adapting playbooks to use collection"
+echo ::group::PLAYBOOK
+find ./playbooks/*/playbook.yml -exec sed -i '/roles:/i collections:' {} \;
+find ./playbooks/*/playbook.yml -exec sed -i '/collections:/a - pulp.pulp_installer' {} \;
+find ./playbooks/*/playbook.yml -exec sed -i 's/collections:/  collections:/g' {} \;
+find ./playbooks/*/playbook.yml -exec sed -i 's/- pulp.pulp_installer/    - pulp.pulp_installer/g' {} \;
```

Comment:
I just ran these 4 commands under `pulp_installer` folder on my laptop.

I see that they add these lines to the example-use playbook, which is symlinked, and the example-source playbook:

+  collections:
+    - pulp.pulp_installer

Would it be possible to just put these in the example playbooks instead? Like under what circumstances would we not want to have these lines? Vagrant? Does running molecule locally (on your laptop) require them as well?

Reply:
I initially thought it would affect vagrant, but I realize now that vagarant uses other playbooks, let me change it